components/match2/wikis/rainbow_six/match_legacy.lua

```diff
@@ -31,8 +31,8 @@ function p.storeMatch(match2)
 end
 
 function p.storeMatchSMW(match, match2)
-	local streams = match.stream or {}
-	if type(streams) == "string" then streams = json.parse(streams) end
+	local streams = json.parseIfString(match.stream or {})
```

Comment:
This change is not mentioned in the description

Reply:
That change is on me, but should indeed be added to the desc

The function `parseIfString` does the exact same thing as the second line 35 used to do. Namely, if the type of the variable is a string, parse it as json, otherwise do nothing. Just some code cleanup. 
```lua
function json.parseIfString(obj)
	if type(obj) == 'string' then
		return json.parse(obj)
	else
		return obj
	end
end

Reply:
Yep, I know. But good practice to mention such things :)